content/blog/2018/06/2018-06-17-running-jenkins-with-java10-11.adoc

```diff
@@ -9,6 +9,11 @@ tags:
 author: oleg_nenashev
 ---
 
+[WARNING]
+--
+Please refer to link:/doc/administration/requirements/jenkins-on-java-11[Running Jenkins on Java 11] documentation to have the up-to-date details on how to run Jenkins on Java 11.
+--
+
```

Comment:
Suggest "Please refer to the link:/doc/administration/requirements/jenkins-on-java-11[Running Jenkins on Java 11] documentation for up-to-date details on running Jenkins on Java 11."